requirements.sonarcov.txt

```diff
@@ -0,0 +1,5 @@
+asgiref==3.7.2
```

Comment:
Installing all the requirements takes ~45s, thus this reduced file to reduce that time as well for the sonar coverage analysis as it doesn't need all the requirements.